docs/pages/versions/v33.0.0/distribution/app-stores.md

```diff
@@ -31,7 +31,14 @@ Try your app on tablets in addition to handsets. Even if you have `ios.supportsT
 - Add a great [icon](../../guides/app-icons/). Icon requirements between iOS and Android differ and are fairly strict, so be sure and familiarize yourself with that guide.
 - Customize your [primaryColor](../../workflow/configuration/#primarycolor).
 - Make sure your app has a valid iOS [Bundle Identifier](../../workflow/configuration/#bundleidentifier) and [Android Package](../../workflow/configuration/#package). Take care in choosing these, as you will not be able to change them later.
-- Use [versionCode](../../workflow/configuration/#versioncode) and [buildNumber](../../workflow/configuration/#buildnumber) to distinguish different binaries of your app.
+
+## Versioning your App 
+
+You'll use the `app.json` file to specify the version of your app, but there are a few different fields each with specific functionality.
+
+- [`version`](../../workflow/configuration/#version) will apply both to iOS and Android. For iOS, this corresponds to `CFBundleShortVersionString`, and for Android this corresponds to `versionName`. This is your user-facing version string for both platforms.
```

Comment:
What is it on web?

Reply:
[Null](https://github.com/expo/expo/blob/master/packages/expo-constants/src/ExponentConstants.web.ts#L71). I added this as part of https://github.com/expo/expo/pull/3894